yarn.lock

```diff
@@ -6614,21 +6624,26 @@ dot-case@^3.0.4:
     no-case "^3.0.4"
     tslib "^2.0.3"
 
+dotenv-expand@10.0.0:
```

Comment:
do we need to change the dependencies here?

Reply:
No, removed them.